### PR TITLE
Tg 2539 isolated load test for create event type

### PR DESCRIPTION
**Pull request by ankit45621**

### CODE REVIEW

File: src/main/java/com/abcfinancial/data/generator/CreateEventCategoryDataGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import org.slf4j.LoggerFactory;
+
+public class CreateEventCategoryDataGeneratorTest
+{
+
+    private static final Logger logger = LoggerFactory.getLogger( CreateEventCategoryDataGeneratorTest.class );
+
+    public static void main( String[] args )
+    {
+
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/generator/CreateEventCategoryDataGeneratorTest.java]: remove extra spaces

File: src/main/java/com/abcfinancial/data/generator/CreateEventTypeDataGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import org.slf4j.LoggerFactory;
+
+public class CreateEventTypeDataGeneratorTest
+{
+
+    private static final Logger logger = LoggerFactory.getLogger( CreateEventTypeDataGeneratorTest.class );
+
+    public static void main( String[] args )
+    {
+
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/generator/CreateEventTypeDataGeneratorTest.java]: same here as above.please remove it from everywhere

File: src/main/java/com/abcfinancial/data/utility/RandomNumberBuilder.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.abcfinancial.data.utility;
+
+public class RandomNumberBuilder
+{
+    private static RandomNumberBuilder countRowSingleton;
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/utility/RandomNumberBuilder.java]: change it to meaningful variable name

File: src/main/java/com/abcfinancial/data/generator/CreateEventCategoryDataGeneratorTest.java (hunk 1 above)
Comment (ankit45621) [reply src/main/java/com/abcfinancial/data/generator/CreateEventCategoryDataGeneratorTest.java]: done

File: src/main/java/com/abcfinancial/data/generator/CreateEventTypeDataGeneratorTest.java (hunk 2 above)
Comment (ankit45621) [reply src/main/java/com/abcfinancial/data/generator/CreateEventTypeDataGeneratorTest.java]: done

File: src/main/java/com/abcfinancial/data/utility/RandomNumberBuilder.java (hunk 3 above)
Comment (ankit45621) [reply src/main/java/com/abcfinancial/data/utility/RandomNumberBuilder.java]: Done

**Pull request merged**